libraries/AP_Mount/AP_Mount_SToRM32.cpp

```diff
@@ -59,31 +57,39 @@ void AP_Mount_SToRM32::update()
         // RC radio manual angle control, but with stabilization from the AHRS
         case MAV_MOUNT_MODE_RC_TARGETING: {
             // update targets using pilot's RC inputs
-            MountTarget rc_target {};
-            if (get_rc_rate_target(rc_target)) {
-                update_angle_target_from_rate(rc_target, _angle_rad);
-            } else if (get_rc_angle_target(rc_target)) {
-                _angle_rad = rc_target;
+            if (get_rc_rate_target(mnt_target.rate_rads)) {
+                mnt_target.target_type = MountTargetType::RATE;
+                update_angle_target_from_rate(mnt_target.rate_rads, mnt_target.angle_rad);
+            } else {
+                mnt_target.target_type = MountTargetType::ANGLE;
+                IGNORE_RETURN(get_rc_angle_target(mnt_target.angle_rad));
             }
+            mnt_target.angle_rad.yaw_is_ef = false;
```

Comment:
setting, the "yaw_is_ef" out here doesn't look correct because this should really depend upon the "Follow/Lock" RC auxiliary function switch position.